week-5/Homework/mandatory/1-exercises/exercises.js

```diff
@@ -0,0 +1,164 @@
+/**
+ *
+ * For each of the names in the array passed into this function
+ *
+ * - Add a <h1> tag to the website containing the name of the person
+ * - Add a <h2> tag to the website containing the job of the person
+ *
+ * All of your HTML should go inside the Div tag with the id "content".
+ *
+ * <div id="content">
+ *      <h1>{Name Here}</h1>
+ *      <h2>{Job Here}</h2>
+ *      .....
+ * </div>
+ */
+function exerciseOne(arrayOfPeople) {
+  let content = document.querySelector("#content");
+  arrayOfPeople.forEach((element) => {
+    hOnes = document.createElement("h1");
+    hOnes.innerHTML = element.name;
+    hTwoes = document.createElement("h2");
+    hTwoes.innerHTML = element.job;
+    let myDiv = document.querySelector("#content");
+    myDiv.appendChild(hOnes);
+    myDiv.appendChild(hTwoes);
+  });
+}
+
+/**
+ *
+ * Create a list of shopping items. You should use an unordered list.
+ *
+ * All of your HTML should go inside the Div tag with the id "content".
+ *
+ */
+function exerciseTwo(shoppingItems) {
+  //Write your code in here
+  ulEl = document.createElement("ul");
+  ulEl.className = "ulstyle";
+  let content = document.querySelector("#content");
+  content.appendChild(ulEl);
+  shoppingItems.forEach((element) => {
+    let liEl = document.createElement("li");
+    liEl.innerHTML = element;
+    ulEl.appendChild(liEl);
+  });
+}
+
```

Comment:
Following on from what I said in my comment on `exerciseOne()`.

In `exerciseTwo()`:

```js
const ulEl = document.createElement(ul);
```

Note that it is OK to use `const` here since `const` does not prevent you from modifying the object, such as to, it simply prevents you assigning another value to that variable identifier (`ulEl`) within the same scope. This might be a good thing since it means if you were to unintentionally do this later in your code then you would get an error.